### ISSUE TITLE
Tooltip format for variable is not working

### ISSUE THREAD

**Issue by alshan** [JetBrains/lets-plot#401]
In the example: https://nbviewer.jupyter.org/github/JetBrains/lets-plot/blob/master/docs/examples/jupyter-notebooks/tooltip_config.ipynb

I've modified one cell as: 

```
# Change format for the "size" aesthetic which is already shown in the tooltip by default.
p + geom_point(shape=21, 
               color='white', 
#                tooltips=layer_tooltips().format('^size', '{.0f} mpg'))
               tooltips=layer_tooltips().format('hwy', '.4f'))
```

as a result the value is not formatted: 
![image](https://user-images.githubusercontent.com/4430922/124183078-ba895500-da85-11eb-83d7-aa9f20c91c63.png)

**Comment by OLarionova-HORIS** [comment]
Currently, aes and var formats are not interchangeable, i.e. var format will not be applied to aes, mapped to this variable (we see aes in the default tooltip). 
The same situation in issue #388.

**Comment by OLarionova-HORIS** [comment]
Fixed: #409.
